packages/litexa/src/parser/skill.coffee

```diff
@@ -478,8 +480,13 @@ class lib.Skill
       # inject code to map typed DB objects to their
       # types from inside this closure
       output.push "__language.dbTypes = {"
-      output.push (for name, def of @dbTypes
-        "  #{name}: #{def.type}").join(',\n')
+      lines = []
+      dbTypeSet = {}
+      for dbType in @dbTypes
+        if dbType.name not of dbTypeSet
+          lines.push "  #{dbType.name}: #{dbType.type}"
+          dbTypeSet[dbType.name] = dbType
+      output.push lines.join ",\n"
```

Comment:
As per above, would need to make sure this is storing the language-specific DB types (if there are any).